### ISSUE TITLE
Exporting Aptly Configuration State

### ISSUE THREAD

**Issue by damageboy** [aptly-dev/aptly#257]
I'd like to somehow "store" the aptly configuration (repos, packages etc.) into source control.

This obviously doesn't exist right now, are there any thoughts about this sort of feature?

**Comment by smira** [comment]
Yes, there were many thoughts about that. Exporting state is not that complex, but importing is much more complicated.

**Comment by damageboy** [comment]
:) I hear you.
Backing up was always an easier task to accomplish without the need to restore...

**Comment by smira** [comment]
So aptly state == DB + files in `pool` + published in `public/` (which could be recreated by re-publishing). DB could be exported more or less easily, it consists of serialized Go structures. Files in the pool - have no idea how to keep them in source control, probably as is.

**Comment by neolynx** [comment]
export/import of repos and mirrors would be a great feature ...
